modules/distributor/forwarder_test.go

```diff
@@ -136,6 +136,8 @@ func TestForwarder_shutdown(t *testing.T) {
 		assert.Equal(t, 0, len(f.queueManagers[tenantID].reqChan))
 	}()
 
+	time.Sleep(time.Second)
```

Comment:
Is this sleep required because we have sleep on line 132?

can we use a waitgroup here, and remove sleep from both places (here and line 132)??